crates/driver/src/infra/config/cli.rs

```diff
@@ -34,6 +36,42 @@ pub struct Args {
     /// The time to allocate to generating quotes, in milliseconds.
     #[clap(long, env, default_value = "5000")]
     pub quote_timeout_ms: u64,
+
+    /// The mempools to use when submitting settlements.
+    #[clap(
+        long,
+        env,
+        default_value = "public",
+        value_enum,
+        use_value_delimiter = true
+    )]
+    pub mempools: Vec<Mempool>,
+
+    /// The Flashbots API URL.
+    #[clap(
+        long,
+        env,
+        use_value_delimiter = true,
+        default_value = "https://rpc.flashbots.net"
+    )]
+    pub flashbots_api_urls: Vec<Url>,
+
+    /// The solver address used to sign transactions. Expects a 20-byte hex
+    /// encoded string. This can't be specified along with --solver-private-key,
+    /// exactly one of them must be specified.
+    #[clap(long, env)]
+    pub solver_address: Option<eth::H160>,
+
+    /// The private key used to sign transactions. Expects a 32-byte hex encoded
+    /// string. This can't be specified along with --solver-address, exactly one
+    /// of them must be specified.
+    #[clap(long, env)]
+    pub solver_private_key: Option<String>,
```

Comment:
The natural Rust type of this would Vec<u8> combined with passing a function to clap to parse it from a hex string similar to what we do with Duation. Or a wrapper type that is parsed that way similar to H160 above. Could be a complete type that validates that it is a correct private key. Then you get better error messages from clap rather than failing when we validate it later.


Not required to merge the PR, just a note.